src/store/ChartStore.js

```diff
@@ -446,6 +446,7 @@ class ChartStore {
     setMainSeriesDisplay(name) {
         // Set display name of main series (to be shown in crosshair tooltip)
         this.stxx.chart.seriesRenderers._main_series.seriesParams[0].display = name;
+        this.stxx.chart.seriesRenderers._main_series.seriesParams[0].field = 'Close';
     }
```

Comment:
@brucebinary After some hours working on this, and according to ChartIQ Team, I use this temporary solution for crosshair

Reply:
@shayanbinary as long as it works. >_< if the solution is temporary, do add a `//TODO...` to indicate that it needs to be removed in the future.